doc/user-guides/full-walkthrough/secure-protect-connect-openshift.md

```diff
@@ -321,71 +321,14 @@ kubectl get dnspolicy ${KUADRANT_GATEWAY_NAME}-dnspolicy -n ${KUADRANT_GATEWAY_N
 while :; do curl -k --write-out '%{http_code}\n' --silent --output /dev/null  "https://api.$KUADRANT_ZONE_ROOT_DOMAIN/cars" | grep -E --color "\b(429)\b|$"; sleep 1; done
 ```
 
-### (Optional) Configure metrics to be scraped from the Gateway instance
```

Comment:
just wondering why removed? isn't it needed for optional dashboard setup?

Reply:
i agree with Craig its a bit jaring but i do agree we need it. Ill add it at the end as a optional